packages/backend-modules/publikator/lib/Derivative/SyntheticReadAloud.ts

```diff
@@ -295,7 +261,7 @@ export const derive = async (
         status: 'Failure',
         result: { error },
         userId: user?.id,
```

Comment:
but we also check that the user is defined there. is there a reason for the double-check